docker-compose.yaml

```diff
@@ -1,4 +1,20 @@
 services:
+  postgres17:
+    image: postgres:17
+    container_name: postgres17
+    command: postgres -c 'max_connections=100'
+    networks:
+      - default
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=password
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 5s
+      timeout: 5s
+      retries: 5
   postgres:
     image: postgres:14
     container_name: postgres
```

Comment:
I edited the `docker-compose` a bit differently than the tests, as I noticed they spun up their own containers dynamically.

If you want, this can just have a single `postgres` entry defaulted to 17. Considering there isn't a `MySQL` here, I think this docker-compose is not meant to be exhaustive